python/aqora/qiskit/client.py

```
@@ -0,0 +1,243 @@
+        page_size: int = 100,
+    ) -> list[Mapping[str, Any]]:
+        after: str | None = None
+        items: list[Mapping[str, Any]] = []
+
+        while True:
+            node = _run_sync(
+                lambda: self._client.send(
+                    PROVIDER_JOB_RESULTS_QUERY,
+                    id=job_id,
+                    first=page_size,
+                    after=after,
```

Ruff (0.15.15)

[warning] Function definition does not bind loop variable `after`

(B023)